README.rst

```diff
@@ -177,11 +173,25 @@ Full Example
 ------------
 
 To see a simple example, you can download the code `from github
-<http://github.com/mkomitee/wsgi-kerberos>`_. It is in the example directory.
+<http://github.com/deshaw/wsgi-kerberos>`_. It is in the example directory.
 
 Changes
 -------
 
+1.1.0
+`````
+-    `hostname` no longer needs to be specified in KerberosAuthMiddleware
+     constructor - any hostname in the keytab will be accepted
+-    Set REMOTE_USER when valid auth is provided, even if not required
+-    Limit the number of bytes read in post data on auth failure to prevent
+     possible DOS attack. New paramter `read_max_on_auth_fail` can be set to
```

Comment:
Typo: “parameter”. Also, maybe “mitigate” rather than “prevent” in the previous line, and maybe write out “denial-of-service” (or at least abbreviate DoS as I think it’s more commonly styled?)